pkg/lib/bundle/generate_test.go

```diff
@@ -207,7 +207,7 @@ func TestGenerateDockerfileFunc(t *testing.T) {
 		"LABEL operators.operatorframework.io.bundle.metadata.v1=%s\n"+
 		"LABEL operators.operatorframework.io.bundle.package.v1=test4\n"+
 		"LABEL operators.operatorframework.io.bundle.channels.v1=test5\n"+
-		"LABEL operators.operatorframework.io.bundle.channel.default.v1=test5\n\n"+
+		"LABEL operators.operatorframework.io.bundle.channel.default.v1=\n\n"+
```

Comment:
Rather than coopting existing tests, can we write an explicit test specifically for this edge case?

Reply:
right, I agree with the previous judgment about the e2e. but the change does alter the outcome for this specific unit test.

Reply:
Ah, this was actually invalid and we weren't catching it. Gotcha